docs/en/latest/FAQ.md

```diff
@@ -401,24 +401,24 @@ In route, we can achieve more condition matching by combining `uri` with `vars`
 
 ## Does the upstream node support configuring the [FQDN](https://en.wikipedia.org/wiki/Fully_qualified_domain_name) address
 
-This is supported. Here is an example where the `FQDN` is `localhost.localdomain`:
+This is supported. Here is an example where the `FQDN` is `httpbin.default.svc.cluster.local`:
```

Comment:
Add some notes to hint readers:

This is supported. Here is an example where the `FQDN` is `httpbin.default.svc.cluster.local` (a Kubernetes Service):

Reply:
updated.

Reply:
Need to remove the repeated `Here is an example where the `FQDN` is `httpbin.default.svc.cluster.local``